### PR TITLE
Polish single file loading API for cc.loader.load

### PR DESCRIPTION
**Pull request by pandamicro**
> Makes sure these boxes are checked before submitting your PR - thank you!
> - [x] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) and end files with a newline
> - [x] Document new code with comments in source code based on [API Docs](https://github.com/cocos-creator/fireball#api-docs)
> - [x] If your pull request has gone "stale", you should **rebase** your work on top of the latest version of the upstream branch.
> - [x] If your commit history is full of small, unimportant commits (such as "fix pep8" or "update tests"), **squash** your commits down to a few, or one, discreet changesets before submitting a pull request.

Re: cocos-creator/fireball#1499

Changes proposed in this pull request:
- 简化单文件加载的回调参数
- 在 AssetLibrary:loadJson 函数的结果中删除 randomUuid

@cocos-creator/engine-admins

@knoxHuang @jareguo 看看现在的 API 使用方式是否能接受

### CODE REVIEW

**Comment by knoxHuang** [comment]
我没意见

File: cocos2d/core/platform/CCAssetLibrary.js [hunk 1]
```diff
@@ -217,6 +215,7 @@ var AssetLibrary = {
                     if (CC_EDITOR || isScene(asset)) {
                         Loader.removeItem(randomUuid);
                     }
+                    delete asset._uuid;
```
Comment (jareguo) [inline cocos2d/core/platform/CCAssetLibrary.js:218]: _uuid 是默认就有的属性，建议不要 delete，设为默认值 '' 即可

**Comment by pandamicro** [comment]
我先合并了，否则有bug。。。有意见我再改

File: cocos2d/core/textures/CCTextureCache.js [hunk 2]
```diff
@@ -318,9 +318,9 @@ game.once(game.EVENT_RENDERER_INITED, function () {
 
             tex = locTexs[url] = new Texture2D();
             tex.url = url;
-            cc.loader.load(url, function (err, items) {
-                if (err) {
-                    return cb && cb.call(target, items.getError(url) || new Error('Unknown error'));
+            cc.loader.load(url, function (err, texture) {
+                if (err || !texture) {
```
Comment (jareguo) [inline cocos2d/core/textures/CCTextureCache.js:322]: 我觉得 cc.loader.load 应该保证返回参数当 !!err，texture 肯定 === null
当 texture 有值，err 肯定 == null
而不是由用户做这种奇怪的判断

File: cocos2d/particle/CCParticleSystem.js [hunk 3]
```diff
@@ -723,10 +723,9 @@ var ParticleSystem = cc.Class({
         var file = this._file;
         if (file) {
             var self = this;
-            cc.loader.load(file, function (err, results) {
-                var content = results.getContent(file);
+            cc.loader.load(file, function (err, content) {
                 if (err || !content) {
```
Comment (jareguo) [inline cocos2d/particle/CCParticleSystem.js:727]: 同上，只要不 err，content 肯定有值，不需要再判断一次 !content

**Comment by pandamicro** [comment]
@jareguo 已完善：https://github.com/cocos-creator/engine/pull/385

**Comment by VisualSJ** [comment]
https://github.com/cocos-creator/fireball/issues/2045

**Pull request merged**